### PR TITLE
[artifactory][artifactory-ha] Allow setting maximum access tomcat connections

### LINKED ISSUE #993: Tomcat artifactory.access.client.max.connections
**Sorry if this feature already exists, I think the way this was managed has changed in Artifactory 7.x**

**Is this a request for help?:**
Yes

This is a
**Feature Request**

I would like to be able to update the Tomcat values for **artifactory.access.client.max.connections**.

As recommended in this document: https://jfrog.com/knowledge-base/how-do-i-tune-artifactory-for-heavy-loads/ it needs to be tuned for heavy loads as below,

Important: When modifying the Access maxThreads, it is required to update the $JFROG_HOME/artifactory/var/etc/artifactory/artifactory.system.properties file with:
artifactory.access.client.max.connections = <VALUE>

**Which chart:**
artifactory-ha

### PR DESCRIPTION
**Pull request by KierranM**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

This PR adds support to both the artifactory and artifactory-ha charts for setting the maximum number of internal tomcat connections for connecting to the access service as per the [tuning for heavy loads](https://jfrog.com/knowledge-base/how-do-i-tune-artifactory-for-heavy-loads) document.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #993

**Special notes for your reviewer**:

I haven't had a chance to test this locally yet, but I've based this on the assumption that if the `corepoolsize` property can be set this way, then this access property should also work.

### CODE REVIEW

File: stable/artifactory/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [10.0.2] - Jul 8, 2020
```
Comment (chukka) [inline stable/artifactory/CHANGELOG.md:4]: @KierranM  Can you please bump the version here , CI is failing

File: stable/artifactory-ha/values.yaml [hunk 2]
```diff
@@ -451,6 +451,9 @@ artifactory:
       extraJavaOpts: >
       {{- with .Values.artifactory.primary.javaOpts }}
         -Dartifactory.async.corePoolSize={{ .corePoolSize }}
+      {{- if .accessClientMaxConnections }}
```
Comment (danielezer) [inline stable/artifactory-ha/values.yaml:454]: Please use with

File: stable/artifactory/CHANGELOG.md (hunk 1 above)
Comment (KierranM) [reply stable/artifactory/CHANGELOG.md]: 🤦 updated

**Comment by eldada** [comment]
> When setting `artifactory.access.client.max.connections`, you also need to set the Access connector maxThreads to the same value.
> We need to couple these parameters. It is also noted in the [blog](https://jfrog.com/knowledge-base/how-do-i-tune-artifactory-for-heavy-loads/) you shared:
> 
> ```
> Important: When modifying the Access maxThreads, it is required to update the $JFROG_HOME/artifactory/var/etc/artifactory/artifactory.system.properties file with:
> artifactory.access.client.max.connections = <VALUE>
> ```

Since we already have 
```
access:
  database:
    maxOpenConnections: 80
  tomcat:
    connector:
      maxThreads: 50
      extraConfig: 'acceptCount="100"'
```
Just set `artifactory.access.client.max.connections={{ .Values.access.tomcat.connector.maxThreads }}` and don't add a new key.

This is BTW - an important fix for scaling. Thx!

**Comment by KierranM** [comment]
> Just set artifactory.access.client.max.connections={{ .Values.access.tomcat.connector.maxThreads }} and don't add a new key.

I hadn't noticed that one! That's a much friendlier method than having to set it twice. I've updated my PR

**Pull request merged**